### ISSUE TITLE
Комментарий к пользовательским сценариям

### ISSUE THREAD

**Issue by aakuptsov** [Ineptum/typist#2]
Пользовательские сценарии отсутствуют. Фиксирую, что дедлайн по пользовательским сценариям пропущен.

**Comment by Ineptum** [comment]
Теперь присутствуют. Их ещё будут рассматривать?01.04.2020, 18:36, "Alexander Kuptsov" <notifications@github.com>:
Пользовательские сценарии отсутствуют. Фиксирую, что дедлайн по пользовательским сценариям пропущен.

**Comment by aakuptsov** [comment]
Да, конечно, но даже беглый просмотр сценариев позволяет сразу сделать вывод, что они не соответствуют требованиям к количеству пунктов в каждом сценарии. Прочти, пожалуйста, ещё раз письмо от меня (оно было относительно давно, но в ЭлЖуре его найти можно по фразе "пользовательские сценарии").

**Comment by Ineptum** [comment]
Прочёл Ваше письмо и отредактировал документ со сценариями: https://github.com/Ineptum/typist/blob/master/hachukaev_user_scripts.md01.04.2020, 21:08, "Alexander Kuptsov" <notifications@github.com>:
Да, конечно, но даже беглый просмотр сценариев позволяет сразу сделать вывод, что они не соответствуют требованиям к количеству пунктов в каждом сценарии. Прочти, пожалуйста, ещё раз письмо от меня (оно было относительно давно, но в ЭлЖуре его найти можно по фразе "пользовательские сценарии").

**Comment by Ineptum** [comment]
Прочёл Ваше письмо и отредактировал документ со сценариями.

**Comment by aakuptsov** [comment]
Необходимо продумать (и добавить) сценарий какого-нибудь соревнования между двумя (хотя бы) участниками. Это может быть соревнование на время или на точность или что-то подобное.

**Comment by Ineptum** [comment]
Соревнование в реальном времени?

**Comment by aakuptsov** [comment]
Да, это было бы интересно и придало бы масштабность твоему проекту.

**Comment by Ineptum** [comment]
Я не совсем уверен, что знаю, как это правильно реализовать и что для этого нужно.

**Comment by aakuptsov** [comment]
Понимаю, поэтому и предлагаю - продумать. Исследуй хотя бы вопрос и, если даже не получится, напиши, почему.

**Comment by Ineptum** [comment]
Хорошо, займусь этим. Спасибо за идею!

**Comment by aakuptsov** [comment]
Всегда рад :)

**Comment by aakuptsov** [comment]
Идеи появились? :)

**Comment by aakuptsov** [comment]
ЗАЧЁТ
